KerbalVR_Mod/KerbalVR/InteractionSystem/KerbalVR_InteractableBehaviour.cs

```diff
@@ -30,19 +30,19 @@ public Hand GrabbedHand
 				{
                     if (lastHand != null && OnRelease != null)
 					{
-						OnRelease.Invoke(lastHand);
+						OnRelease.Invoke(lastHand, lastHand.handType);
 					}
                     lastHand = value;
                     if (value != null && OnGrab != null)
 					{
-                        OnGrab.Invoke(value);
+                        OnGrab.Invoke(value, value.handType);
```

Comment:
space/tab indentation is messed up here; I'd prefer to switch to tabs eventually but the original code used spaces.  Please try to keep new stuff consistent with whatever is around it and eventually we can do a single commit that changes it all.

Reply:
Fixed in 767ad21.